START_HERE.md

```
@@ -0,0 +1,287 @@
+# Create config
+mkdir -p ~/.config/awesome
+cp awesome.config.example ~/.config/awesome/config
+
+# Edit to your preferences
+nano ~/.config/awesome/config
+```
+
+## 💡 Tips
+
+1. **Start with the demo**: `./demo.sh`
+2. **Read QUICKSTART.md**: Quick overview
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)